### PR TITLE
Avoid concurrent time sync of multiple sensors

### PR DESCRIPTION
**Pull request by at-wat**
Automatically change the timing of time sync of urg_stamped nodes in same namespace to avoid stopping scan of all sensors at once.

### CODE REVIEW

File: test/src/sync_timing.cpp [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          .hex_ii_timestamp = false,
+          .angle_resolution = 1440,
+          .angle_min = 0,
+          .angle_max = 1080,
+          .angle_front = 540,
+      };
+
+  ASSERT_NO_FATAL_FAILURE(startSimulator(0, params));
+  ASSERT_NO_FATAL_FAILURE(startSimulator(1, params));
+  ros::Duration(1).sleep();  // Wait starup
```
Comment (f-fl0) [inline test/src/sync_timing.cpp:138]:
```suggestion
  ros::Duration(1).sleep();  // Wait startup
```

File: src/urg_stamped.cpp [hunk 2]
```diff
@@ -837,10 +882,9 @@ UrgStampedNode::UrgStampedNode()
                   boost::arg<2>(),
                   boost::arg<3>()));
 
-  if (clock_estim_interval > 0.0)
+  if (clock_estim_interval_ > ros::Duration(0))
   {
-    timer_delay_estim_ = nh_.createTimer(
-        ros::Duration(clock_estim_interval), &UrgStampedNode::estimateSensorClock, this);
+    next_sync_ = ros::Time::now() + clock_estim_interval_;
```
Comment (f-fl0) [inline src/urg_stamped.cpp:887]: Does this mean that `next_sync_` might be the same time for multiple sensors on start up?

Comment (at-wat) [reply src/urg_stamped.cpp:887]: Yes. Timing will be changed after a collision is detected

**Review by f-fl0 — APPROVED** [review]
LGTM

**Pull request merged**